com.eduke32.EDuke32.yaml

```diff
@@ -0,0 +1,52 @@
+app-id: com.eduke32.EDuke32
+runtime: org.gnome.Platform
```

Comment:
Does it require something from Gnome runtime? If not, better use FD.o runtime.

Reply:
Ah, I think it used to when I made the old PR, but they seem to have removed the menu it had before, and Gnome along with it. Thanks for spotting that.